package.json

```
@@ -5,7 +5,8 @@
   "main": "source/Meadow-Filter.js",
   "scripts": {
     "coverage": "nyc npm run test && nyc report --reporter=lcov",
-    "test": "./node_modules/.bin/mocha --exit -u tdd -R spec"
+    "test": "mocha --ignore \"test/integration/*.js\" --exit -u tdd --recursive",
+    "test:integration": "mocha -u tdd --timeout 20000 --recursive test/integration/jsonFilterTest.js"
```

nit: might not want to limit to a single file
```suggestion
    "test:integration": "mocha -u tdd --timeout 20000 --recursive test/integration/*.js"
```